integration_tests/pages/barcode/createNewContact.ts

```diff
@@ -14,17 +14,55 @@ export default class CreateNewContactPage extends Page {
   }
 
   submitWithValidValues = (): ReviewRecipientsPage => {
+    this.enterAValidPrisonerName()
+    this.typeAheadAValidPrison()
+    return this.submitForm(ReviewRecipientsPage)
+  }
+
+  enterAValidPrisonerName = (): CreateNewContactPage => {
+    this.prisonerNameField().clear()
     this.prisonerNameField().type('Gage Hewitt')
+    return this
+  }
+
+  typeAheadAValidPrison = (): CreateNewContactPage => {
+    this.clearPrisonField()
     this.prisonIdField().type('ashfield')
     this.pressEnterInPrisonIdField()
-    this.submitButton().click()
-    return Page.verifyOnPage(ReviewRecipientsPage)
+    return this
```

Comment:
Returning `this` rather than `Page.verifyOnPage(CreateNewContactPage)` is better in cases like this (where the page hasn't changed) because 

1. the page hasn't changed so we don't need to verify it
2. is (marginally) faster 
3. (most importantly 😄 ) the page class constructor (which `Page.verifyOn` uses) validates that the prisoner name field is auto focussed. This works when the page is loaded and rendered (ie. cases where `Page.verifyOn` is appropriate), but in this case we are simply interacting with the page and form fields. So the focussed field is not the prisoner name field